packages/components/src/Card/Card.css

```diff
@@ -1,10 +1,8 @@
-:root {
+.card {
   --card--accent-color: var(--color-grey);
   --card--base-padding: var(--space-base);
   --card--border: var(--border-base) solid var(--color-border);
-}
```

Comment:
also for cards. might be the case that once we see they continue to work for one we can reasonably assume the rest will be fine.

Reply:
PR to [remove Card usages](https://github.com/GetJobber/Jobber/pull/48156)